### ISSUE TITLE
Firefox 42: TypeError: _currPanBounds is undefined @ photoswipe.js:1088:3

### ISSUE THREAD

**Issue by manuel-84** [dimsemenov/PhotoSwipe#1022]
on other browsers all is working as expected, even on firefox 43.0a2 developer

**Comment by mjau-mjau** [comment]
A bit hard to diagnose with such little info and a minified photoswipe, when the issue is likely related to your custom instantiation of Photoswipe and/or the html structure. This issue is does not occur anywhere else.

If you can reproduce the issue on Codepen, it can be diagnosed properly.

**Comment by manuel-84** [comment]
this is the pen where i can see the same issue: http://codepen.io/anon/pen/NGJrQo

**Comment by mjau-mjau** [comment]
Confirm. Sorry, I really can't identify why you are getting this bug specifically for this example, in Firefox.

**Comment by dimsemenov** [comment]
What exactly to do to replicate the issue? And on what exactly OS do you test it? For some reason I'm not getting any errors in FF42, not on Mac, not on Windows.

**Comment by manuel-84** [comment]
On Windows 8.1 and Firefox 42 when I click the button to load PhotoSwipe it loads the image but the interface is invisible and inactive, I can't zoom or close

**Comment by mjau-mjau** [comment]
I can reproduce the error on Mac/Firefox also. It happens randomly, and I can't see what the cause could be. I suspect it's related to multiple instances of Photoswipe.

http://codepen.io/anon/pen/NGJrQo
![codepen](http://d.pr/i/i0Hy+)

**Comment by manuel-84** [comment]
I tried with Firefox 44 (dev. edition) and seems working fine

**Comment by dimsemenov** [comment]
Please check if you can replicate issue on this pen http://codepen.io/dimsemenov/full/Oyqwav/ , maybe it's related to how images are cached in FF.

**Comment by mjau-mjau** [comment]
> Please check if you can replicate issue on this pen http://codepen.io/dimsemenov/full/Oyqwav/ , maybe it's related to how images are cached in FF.

I can't seem to reproduce the error in the above pen. Interesting.

**Comment by manuel-84** [comment]
@dimsemenov yes, i can replicate the issue in your pen and i also tried to replace the image links with placekittens+randomquery

**Comment by smm1378** [comment]
very old ! but for any one need :
problem will solve with showAnimationDuration:0 and hideAnimationDuration:0 in options . example : 
`
        let options = {
                index: 0,
                showAnimationDuration:0,
                hideAnimationDuration:0,
                modal: true
        };
        // Initializes and opens PhotoSwipe
        Gallery_Core = new PhotoSwipe(pswpElement, PhotoSwipeUI_Default, items, options);`
